File: inhertance/src/CurvedBanner.java
package business;

 public class CurvedBanner extends Banner {

	 private double radius;

	 public CurvedBanner(){
		 radius = 0.5;
	 }

	 public CurvedBanner(int w, int h, int r) {
		 super(w, h);
		 radius = r;
	 }

	 public double getRadius(){
		 return radius;
	 }

	 public void setRadius(double r) {
		 radius = r;
	 }

	 public double getArea() {
		 double area = super.getArea();
		 return area - (3.14 * radius);
	 }





 }

File: Assignment3/GeoLib/CurvedBanner.cs
namespace GeoLib;

public class CurvedBanner : Banner 
{
    private float radius;

    public CurvedBanner(float w, float h, float r):base(w,h)
    {
      radius = r;  
    }

    public CurvedBanner():base()
    {
        radius = 0.1f;
    }

     public float Radius
     {
        get { return radius;}
        set {  radius = value;}
     }

     public override float Area()
     {
        float a = base.Area();
        return a - 0.86f * radius * radius;
     }
}
